migrations/2023-03-17-132427_create_twabs/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE twabs (
```

Comment:
will this only ever run on mainnet or do we want to track network here as well?

Reply:
good Q! I imagine we would have a dev env and db if we decide to track it there 🤔 . But could be simpler to combine